env/work/srw_python/SRWLIB_Example17.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+try:
```

Comment:
This example is to reproduce calculation with the optical mask (code from Metrology group).